dpe/src/tci.rs

```diff
@@ -1,10 +1,10 @@
 // Licensed under the Apache-2.0 license.
 use crate::DPE_PROFILE;
 use zerocopy::{AsBytes, FromBytes};
-use zeroize::Zeroize;
+use zeroize::ZeroizeOnDrop;
 
 #[repr(C, align(4))]
-#[derive(Default, Copy, Clone, AsBytes, FromBytes, PartialEq, Eq, Zeroize)]
+#[derive(Default, Clone, AsBytes, FromBytes, PartialEq, Eq, ZeroizeOnDrop)]
 pub struct TciNodeData {
```

Comment:
I don't think there's anything sensitive in this type. I think the most important type to ZeroizeOnDrop is CryptoBuf, which I believe we already do.